docs/content/en/main/concepts/investigations/edge-multi-cluster.md

```diff
@@ -0,0 +1,82 @@
+---
+title: "Edge multi-cluster"
+linkTitle: "Edge multi-cluster"
+weight: 1
+description: >
+  Investigation of edge multi-cluster
+---
+
+Kubernetes offers a widely adopted, state-based management platform to declaratively develop and operate containerized workloads. Kubernetes allows a 
+developer to create deployments that are node-agnostic and can be constrained to operate in places where certain conditions are met. Developers and 
+Operators understand that pods will be rescheduled to execute on nodes wherever constraints allow them to run. This approach has satisfied the large
+   majority of use cases in the cloud native community.
+
+There is a growing community of cloud native developers and operators that require more fine-grained control
+ over the distribution of pods due to the nature of their business. Telecommunications, industrial manufacturers, and automotive
+  companies would like to rely on Kubernetes state-based management but they need more flexibility and specificity to accomodate for requirements like
+  large numbers of edge locations, disconnected and autonomous operation, and constrained cluster size. In the edge multi-cluster usecase, there are
+  situations when pods must run on a specific edge location on a specific node. Managing a set of labels, taints, and tolerations to cover all edge 
+  locations is too difficult.
+
+A key area of investigation for `kcp` is exploring large-scale deployment of workloads that can operate under constraints and
+ requirements unlike those accomodated by upstream Kubernetes or KCP's Transparent MultiCluster (TMC). Ideally, we want declaritively defined workloads 
+ to operate across a flexible topology combining KCP workspaces and Kubernetes cluster namespaces to achieve greater scale and control over placement. 
+ Deployment of workloads across many thousands of destinations that include namespaced and non-namespaced objects, receiving state and reporting
+  status (grouped and individually), require new interfaces based on (and extending) KCP primitives.
+
+## Goal: Placement strategies
```

Comment:
A. Why is this 2nd-level heading a "goal" and the next two _not_ a goal?
B. We should give peer handling of all the distinctive concerns of edge-MC.